### PR TITLE
add Single Cell shed category

### PR DESCRIPTION
**Pull request by pavanvidem**

### CODE REVIEW

**Comment by SaimMomin12** [comment]
Strange that R linting fails on the following files:

`music_deconvolution/scripts/estimateprops.R`
`music_deconvolution/scripts/dendrogram.R` and
`music_deconvolution/scripts/compare.R` 

which surprisingly are not part of your PR? or?

File: tools/music_deconvolution/.shed.yml [hunk 1]
```diff
@@ -8,6 +8,7 @@ remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/master/to
 type: unrestricted
 categories:
 - Transcriptomics
+- Single Cell
```
Comment (bgruening) [inline tools/music_deconvolution/.shed.yml:11]: @SaimMomin12 it is ....

Comment (SaimMomin12) [reply tools/music_deconvolution/.shed.yml:11]: Ah! Got it..

**Comment by pavanvidem** [comment]
is there anything that I should fix here?

**Pull request merged**